src/requestApproveTemplate.ts

```diff
@@ -0,0 +1,53 @@
+import { S3 } from 'aws-sdk';
+import * as fs from 'fs';
+import * as cli from 'cli-color';
+import { Md5 } from 'ts-md5/dist/md5';
+
+import { Arguments } from 'yargs';
+import { loadStackArgs, parseS3HttpUrl } from './cfn/index';
+import configureAWS from './configureAWS';
+
+export async function requestApproveTemplate(argv: Arguments): Promise<number> {
+  const stackArgs = await loadStackArgs(argv);
+  const s3 = new S3();
+
+  await configureAWS(stackArgs.Profile, stackArgs.Region);
+
+  const cfnTemplate = await fs.readFileSync(stackArgs.Template);
+  const cfnTemplateFileName = stackArgs.Template.match(/[a-zA-Z-_0-9]*\.yaml/)![0];
```

Comment:
`.yaml` isn't the only supported extension (could be `.yml` or `.json`). I'm also not sure what or why you're matching on something  here - it looks like this is equivalent to `path.basename`? It is only used in the log statement below, where I think you should just use the whole template path or maybe the absolute path to the template.

Reply:
This is only used for logging. I can drop this line to make it simpler